sdks/python/tox.ini

```diff
@@ -244,16 +244,6 @@ commands =
   yapf --version
   time yapf --diff --parallel --recursive apache_beam
 
-[testenv:py3-dependency-check]
-# TODO(https://github.com/apache/beam/issues/20337): botocore, a part of [aws], wants docutils<0.16, but Sphinx
-# pulls in the latest docutils. Uncomment this line once botocore does not
-# conflict with Sphinx:
-# extras = docs,test,gcp,aws,interactive,interactive_test
-extras = test,gcp,aws,dataframe,interactive,interactive_test
-passenv = WORKSPACE
-commands =
-  time {toxinidir}/scripts/run_dependency_check.sh
```

Comment:
We should be able to get rid of `run_dependency_check.sh`, right? https://github.com/apache/beam/blob/8ec667c8a0b746849eed6b0199e41070c49a110b/sdks/python/scripts/run_dependency_check.sh#L4

Reply:
Didn't realize there was another script in the Python SDK, I had already cleared out the one in the .test-infra directory